src/main/java/pages/CataloguePage.java

```diff
@@ -1,59 +1,178 @@
 package pages;
 
+import org.openqa.selenium.By;
 import org.openqa.selenium.NoSuchElementException;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
 import java.util.List;
 
-public class CataloguePage extends BasePage {
+public class CataloguePage extends HeaderAnyPage {
```

Comment:
Why do we need to extend `HeaderAnyPage` class here? BTW please, rename `HeaderAnyPage` to `BaseHeader`

Reply:
ok. to make possible to work with header (serarch and user account actions) from any page.